company/team/index.md

```diff
@@ -473,6 +473,15 @@ To add yourself to this page, copy the following template, paste it at the end o
 - [oconvey@sourcegraph.com](mailto:oconvey@sourcegraph.com), [LinkedIn](https://www.linkedin.com/in/owenconvey/)
 - Owen was born and raised in England and has been residing in Barcelona since 2012.  He spends a lot of time watching Rugby, playing with his young daughter and tinkering with his laptop.  Prior to Sourcegraph, he was a manager at Help Scout and loves empowering and motivating autonomous engineering teams.  His Achilles' heel is most definitely pizza.
 
+## Virginia Ulrich (she/her)
+
+- Head of Customer Support
+- Portland, Oregon, United States 🇺🇸
+- GitHub: virginiaulrich
+- [virginia@sourcegraph.com](mailto:virginia@sourcegraph.com), [LinkedIn](https://www.linkedin.com/in/virginiaulrich/) 
+- Name pronunciation: vr ji nyuh  oo l ri ch
+- Virginia lives with her partner (Vinh) and two cats, Little Miss Gable (tuxedo) and Hercules Mulligan (orange tabby). She derives joy from the tiny beautiful moments of human connection, hikes across the city and out in nature, and cooking for others. She studied literature and discovered her love of computer science quite by accident. With many yaers in customer service in various forms, Virginia can geek out for hours about the philosophy of 👍 customer support, the importance of compassion > empathy, and all things involving people helping other people.
```

Comment:
Typo in years `yaers`.  Unless this is specific Portland slang?

Reply:
Ha! Thank you!

Reply:
I updated it :)